.github/workflow/aggregate.yaml

```diff
@@ -0,0 +1,42 @@
+
+name: Aggregate daily data
+on: 
+  schedule:
+    # at 4AM, 10AM, 4PM, and  10PM UTC
+    # at 12AM, 6AM, 12PM, and  6PM EST
+    - cron '0 4/6 * * *'
+
+jobs:
+  test-and-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set NodeJS version
+        id: node_version
+        run: echo ::set-output name=NODE_VERSION::$(cat .nvmrc)
+
+      - name: Set Up NodeJS
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ steps.node_version.outputs.NODE_VERSION }}
+          cache: "yarn"
+
+      - name: Setup dependencies
+        run: yarn install --immutable --check-cache
+        run: yarn add ardrive-cli
+
+      - name: Write wallet
+        run: echo -n '${{ secrets.WALLET_FILE }}' | base64 --decode > /tmp/wallet
+
+      - name: Build
+        run: yarn build
+
+
+      - name: Fetch previous output
+        run: ardrive download-file -f 7fa5d4e3-0087-422a-acb3-2e481d98d08b
+
+      - name: Run
+        run: yarn node aggregate
+
+      - name: Publish output
+        run: ardrive upload-file --local-path ./daily_output.json -F "d62a47c3-0b9d-4442-ac72-252a239d0469" -w /tmp/wallet --no-bundle --upsert --dry-run
```

Comment:
Are we merging this with `--dry-run`?

Reply:
I want to test it with what's at #12, which changes the action as well